### DISCUSSION TITLE
Porto Alegre - Evento para Devs interessados em trabalho remoto

### DISCUSSION THREAD

**Discussion by tiagochi** [frontendbr/forum#1448] [Dicas]
Oi Pessoal,  sou o Community Leader do Toptal em Porto Alegre e gostaria de convidar os que são daqui (ou estão por aqui) para um happy hour onde vocês podem conhecer quem trabalha com o Toptal, fazer perguntas e é claro, tomar uma cerveja.

Pra quem não sabe, o Toptal é uma rede de developers e designers que trabalham remotamente para vários clientes em todo o mundo. Saiba mais em www.toptal.com

Segue o link pra se inscrever no evento: https://www.toptal.com/community/events/2019-06-18/toptal-networking-event-porto-alegre

**Comment by pietrobelluno** [comment]
@tiagochi 
Opa, tudo certo? Sou de Caxias do Sul, tenho interesse em trabalho remoto, teria algum evento futuro aqui na região, ou eu poderia te adicionar para conversarmos sobre isso?

**Comment by adeonir** [comment]
Eu pude ir num evento da Toptal em 2017 e achei muito bom.
Ano passado não pude ir e esse ano também não, principalmente por ser no meio da semana.

Sou de Carlos Barbosa e seria interessante se o evento fosse numa sexta ou num sábado, ou até mesmo aqui na serra ;)

**Comment by tiagochi** [comment]
@pietrobp
Pode me adicionar sim. Pode me mandar e-mail também no tiago.chilanti @toptal.com

**Comment by tiagochi** [comment]
@adeonir
Legal!
Estamos pensando em fazer o próximo evento em algum lugar da Serra!! Aguarde!

**Comment by pietrobelluno** [comment]
@tiagochi mandei um e-mail pra ti la !
